.github/workflows/prepare-release.yml

```diff
@@ -15,6 +18,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          ref: ${{ inputs.branch }}
 
       - name: Update translations from weblate
         uses: ./.github/actions/weblate-pull-translations
```

Comment:
This will replace released translations with translations for code from main branch. I'm not sure it is a good idea. It will replace not only *.po files, but also *.pot files, meaning that messages removed from main branch would disappear their translations from stable branch. Unless we are going to maintain stable branches in dnf5-l10n repository too, I would disable this step for non-main branch.